conn2res/iodata.py

```diff
@@ -163,14 +176,13 @@ def fetch_dataset(task, n_trials=100, *args, **kwargs):
 def create_nativet_dataset(task, tau_max=20, **kwargs):
 
     if task == 'MemoryCapacity':
-        x = np.random.uniform(-1, 1, (1000+tau_max))[:,np.newaxis]
-        y = np.hstack([x[tau_max-tau:-tau] for tau in range(1,tau_max+1)])
-    
-    return x, y
+        x = np.random.uniform(-1, 1, (1000+tau_max))[:, np.newaxis]
+        y = np.hstack([x[tau_max-tau:-tau] for tau in range(1, tau_max+1)])
 
+    return x, y
 
-def split_dataset(data, frac_train=0.7):
 
+def split_dataset(*args, frac_train=0.7, axis=0):

```

Comment:
I updated this function so that I can splits datasets in parallel, i.e. it can operate with multiple number of input data (e.g. x, y passed as *args).

Also, I defined an axis input argument as in some cases after data splitting we might need to concatenate along the second axis (i.e. axis=1).